### ISSUE TITLE
Create AUR package

### ISSUE THREAD

**Issue by TheAvidDev** [TheAvidDev/sway-borders#5]
I use Arch so I'll make an AUR package (`sway-borders-git`) for this fork. I'm not sure how releases will work, but when it comes to making them I'll try to make a `sway-borders` package as well.

**Comment by hrfee** [comment]
Are you planning for this to be installable alongside other versions of sway without conflicting?

**Comment by TheAvidDev** [comment]
No. This will conflict with the `sway` package the same way `sway-git` conflicts with, and provides, `sway` as well.

If you would like to have multiple versions, clone the various versions multiple times and build them separately, but don't install. Then you can run the versions directly from the output executable such as `./build/sway/sway`.

**Comment by hrfee** [comment]
Ah ok, that makes more sense.

**Comment by chaitunya** [comment]
@TheAvidDev Since I haven't seen any more information about the AUR package, I've tried my hand at creating a PKGBUILD. It's pretty simple--I based it off of the sway-git AUR package:
```
pkgname=sway-borders-git
_pkgname=sway-borders
pkgver=r6518.0dc19521
pkgrel=1
license=("MIT")
pkgdesc="i3-compatible Wayland compositor"
makedepends=("meson" "git" "scdoc" "wayland-protocols" "ninja")
depends=(
	"json-c" "pcre" "wlroots-git" "cairo" "pango" "gdk-pixbuf2" "pam"
	"xorg-server-xwayland" "swaybg-git"
)
optdepends=(
	"alacritty: Default terminal emulator."
	"dmenu: Default for launching applications."
	"i3status: To display system information with a bar."
	"swayidle-git: Idle management daemon for Sway."
	"swaylock-git: Screen locker for Sway."
)
arch=("i686" "x86_64")
url="https://swaywm.org"
source=("${pkgname%-*}::git+https://github.com/TheAvidDev/sway-borders.git")
sha512sums=('SKIP')
provides=("sway")
conflicts=("sway")
options=(debug !strip)

pkgver() {
	cd "$_pkgname"
	printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
}

prepare() {
	cd "$_pkgname"
}

build() {
	cd "$_pkgname"
	meson \
		-Dwerror=false \
		--prefix /usr \
		"$srcdir/build"
	ninja -C "$srcdir/build"
}

package() {
	cd "$_pkgname"
	DESTDIR="$pkgdir" ninja -C "$srcdir/build" install
}
```

It worked for me. If you would like, either you or I can submit it to the AUR.

**Comment by TheAvidDev** [comment]
Thanks for making this, it looks basically exactly like what I had in mind and already made. My intention was to finish the other PRs, namely #3, #6, and #10 before putting this on the AUR.

Let me know if you would rather have this in the AUR before closing those PRs, in which case I'll do that instead.

**Comment by chaitunya** [comment]
That makes sense. I agree that you should push the package to the AUR once those issues (I think you meant issues, not PRs) are closed.

**Comment by TheAvidDev** [comment]
Ah yep, wrong wording. Feel free to work on them though :smile:

**Comment by spikecodes** [comment]
I would love to see this awesome project on the AUR!

**Comment by TheAvidDev** [comment]
Great news, #17 has been opened to close #6. Please test it out and let me know if you find any issues. I'll leave it up for the next few days, while trying to fix #3 and #10, but will create an AUR package regardless.

**Comment by TheAvidDev** [comment]
Alright, the package is up! Please test it out and let me know if there's any issues.

`sway-borders-git`: https://aur.archlinux.org/packages/sway-borders-git/
